test/slick.test.js

```diff
@@ -1,7 +1,51 @@
-describe('Slick test', function() {
-  
-  it('should render slides with dots', function() {
-    
+import Slider from '../src/slider.jsx';
+import props from '../src/default-props.js';
+
+const {
+  renderIntoDocument,
+  findRenderedComponentWithType,
+  scryRenderedDOMComponentsWithClass
+} = React.addons.TestUtils;
+
+describe('Slick test', () => {
+
+  context('when rendering with default slide class', () => {
+    let slides;
+
+    before(() => {
+      const component = renderIntoDocument(
+        <Slider {...props}>
+          <div><h1>Something</h1></div>
+        </Slider>
+      );
+      const slider = findRenderedComponentWithType(component, Slider);
+      slides = scryRenderedDOMComponentsWithClass(slider, 'slick-slide');
+    });
+
+    it('renders slides with the default class', () => slides.should.not.be.empty());
+
+    it('renders the correct amount of slides', () => slides.should.length(3));
   });
-  
-});
\ No newline at end of file
+
+  context('when rendering with custom slide class', () => {
+    let slides;
+
+    before(() => {
+      const slidesClass = 'customClass';
+      const component = renderIntoDocument(
+        <Slider {...props} slidesClass={slidesClass}>
+          <div><p>Random text</p></div>
```

Comment:
You can  add the custom class in slides

For example
```
<div className='customClass'><p>Random text</p></div>
```
React slick will add remaining classes while keeping customClass
```
<div className='slick-slide slick-active customClass'><p>Random text</p></div>
```